public/v4/apps/baserow.yml

```diff
@@ -0,0 +1,90 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        image: tiredofit/baserow:$$cap_baserow_tag
```

Comment:
I am not sure how up to date this unofficial Baserow Docker image is. It seems the latest was pushed at least two months ago, but we released a new version of Baserow weeks ago. We're working on official Docker images, but we not completely there yet.

Reply:
When it's complete, just leave another comment here, then I will update the pull request.

Reply:
@bram2w please be noted that to work best with self-hosted env, you should support multiple paths (/api, /media, ...) instead of multiple subdomains (api.something.etc, media.something.etc)

Reply:
I am aware of that. For the official Docker images we are going to split the backend and the web-frontend into two different images, but I think another image that combines everything and uses path routing would be great. We are already doing that for the Cloudron and Heroku integration.